### PR TITLE
Support MySQL, see b/64102112

### PR DESCRIPTION
**Pull request by grant**
See: https://github.com/docker-library/mysql/issues/186
b/64102112

### CODE REVIEW

**Comment by ahmetb** [comment]
cc: @sanche is the new maintainer.

File: wordpress-persistent-disks/mysql.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ spec:
       containers:
         - image: mysql:5.6
           name: mysql
+          args:
+            - "--ignore-db-dir=lost+found"
```
Comment (ahmetb) [inline wordpress-persistent-disks/mysql.yaml]: this fix is only necessary if you're gonna change image to mysql:5.7.

this manifest actually works just fine with the currently listed image. the bug only happens when people try to use a newer image.

Comment (grant) [reply wordpress-persistent-disks/mysql.yaml]: OK. I've changed the image to 5.7.

**Comment by ahmetb** [comment]
The bug  had some discussion on this. We can straight move to :8, if it works.

**Comment by grant** [comment]
> cc: @sanche is the new maintainer.

I believe you mean sanche@ or @DanSanche.

---

@DanSanche PTAL and suggest changes or merge.

**Comment by DanSanche** [comment]
I agree with Ahmet that if we're going to update MySQL, we should try to update straight to :8. Has anyone tried it to see if it works? Any known issues?

I can try to run through it myself, but it might be in a couple days

**Comment by DanSanche** [comment]
I just tried out v8. It looks like lost+found isn't an issue, but we do need to change the auth plugin:

```
        - image: mysql:8
          name: mysql
          args:
            - "--default-authentication-plugin=mysql_native_password"
```

**Comment by grant** [comment]
Hey @DanSanche, can you recommend a way for me to make progress on this PR?
LMK what you'd like to do.

**Comment by DanSanche** [comment]
I think your last commit should be all we need. I'll run through it again today to verify

**Pull request merged**